app/src/main/java/com/duckduckgo/app/browser/PulseAnimation.kt

```diff
@@ -92,6 +87,30 @@ class PulseAnimation(private val lifecycleOwner: LifecycleOwner) : LifecycleObse
         }
     }
 
+    private fun getPulseObjectAnimator(view: View): ObjectAnimator {
+        val width = view.width
+        val height = view.height
+        return if (width != height) {
```

Comment:
This fixes a problem when showing the pulse animation on view that are not square. I've noticed this problem when promoting the browser options menu.